index.html

```diff
@@ -27,5 +27,118 @@ <h2 class="title-2">Software developer</h2>
             </p>
         </div>
     </section>
+    <hr class="hrbar">
+    <section class="feature-section">
+        <div class="features">
+            <div class="lang">
+                <div class="feature-title">
+                    <h6>Language</h6>
+                </div>
+                <div class="feature-detail">
+                    <ul class="featureslist"> 
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>Html</li>
+                        <li>Css</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="framework">
+                <div class="feature-title">
+                    <h6>Frameworks</h6>
+                </div>
+                <div class="feature-detail">
+                    <ul class="featureslist"> 
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="lang">
+                <div class="feature-title">
+                    <h6>Skills</h6>
+                </div>
+                <div class="feature-detail">
+                    <ul class="featureslist"> 
+                        <li>Codekit</li>
+                        <li>Github</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+    <hr class="hrbar">
+    <div class="socialicons">
+        <i class="fab fa-github"></i>
+        <i class="fab fa-linkedin-in"></i>
+        <i class="fab fa-twitter"></i>
+        <i class="fab fa-facebook"></i>
+    </div>
+    <section class="recnt-projects">
+        <div class="project-setion">
+            <i class="fa fa-arrow-down work-icon"></i>
+            <h6 class="work-title">Work</h6>
+        </div>
```

Comment:
I think we have to stick to the design as much as possible (e.g., font, colors, images, tests, margins) using the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2346).

Your design:
![image](https://user-images.githubusercontent.com/60292761/148612927-a2f1e07c-3e86-4b3f-89af-783963efa7f8.png)
Figma design:
![image](https://user-images.githubusercontent.com/60292761/148613010-754e7038-a625-4e77-9378-e0544ad8556c.png)

Reply:
@aliabbani , Thanks for your review, I have solved this issue and now it is the same as Figma design